feature/experimental/backup_nhg/ate_tests/backup_nhg_test/backup_nhg_test.go

```diff
@@ -127,6 +130,24 @@ func TestDirectBackupNexthopGroup(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 	configureDUT(t, dut)
 	configureNetworkInstance(t, dut)
+	// For interface configuration, prefer config Vrf first then the IP address
+	if deviations.InterfaceConfigVRFBeforeAddress(dut) {
+		configureDUT(t, dut)
+	}
+
+	if deviations.BackupNHGRequiresVrfWithDecap(dut) {
+		d := &oc.Root{}
+		ni := d.GetOrCreateNetworkInstance(vrfA)
```

Comment:
Typically the vrf selection policy is only created in the DEFAULT NI.  did this work in EOS ?

Reply:
It is working. Is there a way to do this using Default VRF without changing the Test plan?